theme6/source/stylesheets/site.css.scss

```diff
@@ -1,159 +1,222 @@
+  width: 100%;
+  margin-left: 0px;
+  padding-left: 0px;
+}
+
+.information li {
+  display: flex;
+  justify-content: space-between;
+}
+
+.social-media-container {
+  width: 100%;
+  display: flex;
+}
+
+.social-media-box {
+  flex: 1;
+  display: flex;
+  justify-content: flex-start;
+  transform: rotateY(100px);
+}
+
+.logo {
+  display: flex;
+  width: 100px;
+  justify-content: center;
+  align-content: center;
+}
+
+.social-media {
+  color: #72808C;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 18px;
+  line-height: 21px;
+}
+
+.sm {
+  border: 2px solid #72808c;
+  border-radius: 3px;
+  height: 35px;
+  width: 93px;
+}
+
+.fb {
+  background: url("/images/fb.svg");
+  background-position: center center;
+  background-repeat: no-repeat;
+}
+
+.twt {
+  background: url("/images/twt.svg");
+  background-position: center center;
+  background-repeat: no-repeat;
+  margin-left: 15px;
+  margin-right: 15px;
+}
+
+.g {
+  background: url("/images/g+.svg");
+  background-position: center center;
+  background-repeat: no-repeat;
+}
+
+.newsletter {
+  display: flex;
+  color: #72808c;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 18px;
+  text-align: right;
+  align-items: flex-end;
+  line-height: 21px;
+  text-align: center;
+  flex:1;
+  flex-direction: column;
+}
+
+.newsletter .box-newsletter {justify-content: center;}
+
+.newsletter input {
+  width: 300px;
+  height: 20px;
+  font-size: 15px;
+}
+
+.social-media-box {
+  align-content: center;
+  text-align: center;
+  width: 350px;
+}
+
+.social-media-icon-group button {cursor: pointer;}
+
+.social-media-container {
+  align-items: center;
+  display: inline-flex;
+  justify-content: center;
+  width: 100%;
+}
+
+.information a {
+  color: #72808C;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 18px;
+  line-height: 21px;
+  text-decoration: none;
+}
+
+.author {
+  color: #72808c;
+  display: block;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 16px;
+  line-height: 18px;
+  text-align: center;
+}
+
+@media screen and (max-width: 1024px){
+  .information a{ font-size: 14px}
+
+
+  .sm {
+    border: 2px solid #72808c;
+    border-radius: 3px;
+    height: 25px;
+    width: 60px;
+  }
+
+  .newsletter input {
+    width: 200px;
+    height: 15px;
+    font-size: 12px;
+  }
+
+  .newsletter p {font-size: 15px;}
+
+}
+
+
+@media screen and (max-width: 768px) {
+  .information {display: none;}
+
+  .logo { display: none;}
+
+  .social-media-container {
+    display:block;
+    width: 100%;
+  }
+
+  .social-media-box {
+    width: 100%;
+    display: block;
+  justify-content: center;
+  height: auto;
+}
+
+  .social-media p {
+    width: 100%;
+    text-align: center;
+    font-size: 14px;
+  }
+
+  .social-media-icon-group {
+    display: inline-flex;
+    justify-content: center;
+  }
+
+  .twt {
+    margin-left: 3px;
+    margin-right: 3px;
+  }
+
+  .newsletter {
+    display: block;
+    justify-content: center;
```

Comment:
Properties should be ordered display, display, justify-content, padding-bottom, text-align, transform, width